translations/it/pack/tfa/tof.json

```diff
@@ -8,13 +8,13 @@
     {
         "code": "04104",
         "name": "Mira Precisa",
-        "text": "Veloce. Gioca solo quando attivi la capacità <b>Combatti</b> su un supporto [[Arma da Fuoco]] o [[Distanza]].\nQuesto attacco può bersagliare 1 nemico in un luogo collegato. Ignora le parole chiave Sfuggente e Ritorsione in questo attacco. Se hai successo attaccando un nemico non impegnato con te, infliggi +1 danno in questo attacco.",
+        "text": "Veloce. Gioca solo quando attivi la capacità <b>Combatti</b> su un supporto [[Arma]] [[da]] [[Fuoco]] o [[Distanza]].\nQuesto attacco può bersagliare 1 nemico in un luogo collegato. Ignora le parole chiave Sfuggente e Ritorsione in questo attacco. Se hai successo attaccando un nemico non impegnato con te, infliggi +1 danno in questo attacco.",
```

Comment:
I think this might want to remain as `[[Arma da Fuoco]]` -- was it breaking the formatting somewhere?  I believe the intent was for each of these to map to a single 'Trait' (perhaps to enable hyperlinking or something someday?)

Reply:
It was shown on the site as [[Arma da fuco]] without the italics/bold and with the visible brackets.
I was thinking it could be because it was coded for a single word, hence I tried putting every word in brackets.
I know nothing about coding though, I'm just trying to keep things together in an elegant way :D

Reply:
Ahh, yes the code in arkhamdb was assuming no spaces.

I submitted a PR to address this issue to the arkhamdb repo.

Reply:
Oh by the way, if you can, let me know how I can do a single pull request with multiple files.
I changed a bunch of things over the whole Italian section and I created 12 requests. Allen's gonna kill me. :D